ush/forecast_postdet.sh

```diff
@@ -134,7 +129,8 @@ EOF
     #.............................
 
   else ## cold start
-    for file in $(ls $memdir/INPUT/*.nc); do
+    ICSDIR=${ICSDIR:-${memdir}/INPUT}         # cold start initial conditions
```

Comment:
I'm not sure if this is going to work or not. `config.base` sets `ICSDIR` with a value provided by `setup_exp.py`, but if one isn't provided, the value is `None`, to the fallback would never be used here. This will cause issues if people stage ICs manually. Plus, `setup_exp.py` should've already copied the ICs to the `INPUT` directory already.

Reply:
The changes is to allow users can use different ICSDIR, otherwise, we mush put the ICS into the ${memdir}/INPUT.  Or I can use **ICSIDR_ATM** to be consistent with **RSTDIR_ATM**. How do you think?

Reply:
If user does not provide the ICSDIR, then ICSDIR will use default value: ${memdir}/INPUT

Reply:
I tested the forecast in GEFS repo to set ICSDIR, and it works well.

Reply:
@aerorahul @WalterKolczynski-NOAA I just removed the ICSDIR.